### DISCUSSION TITLE
Which repository for Team Tickets

### DISCUSSION THREAD

**Discussion by PhillipNalesny** [eclipse-tractusx/sig-release#362] [Q&A] [answered]
Hello, 

in which repository do I create tickets for which purpose? I want to organize my team's tasks, e.g. for developing a FOSS application. I've created this project: https://github.com/orgs/eclipse-tractusx/projects/42/views/2
When I want to create an issue I have to select a rep. Here is a screen shot
![grafik](https://github.com/eclipse-tractusx/sig-release/assets/146107795/35f7206d-623c-43d3-8838-50535550916f)

Thank you very much in advance for your help! :) Best regards Phil

**Comment by FaGru3n** [comment, marked as answer]
Hi @PhillipNalesny, 

warm welcome from me. 

To your question ... to which project / team / component does the issue belong to... ?

There are multible repositories, like for some teams like sig-release or sig-infra
but also for hole components like edc, miw, puris, portal, ....
https://github.com/orgs/eclipse-tractusx/repositories?type=all

So basic question do you have a repo where you can raise your issues against it? If not maybe you could think of creating one.
But needs all information that are described here: https://eclipse-tractusx.github.io/docs/oss/issues

**Reply by PhillipNalesny** [reply]
Hey @FaGru3n, thanks for your quick answer. 

So if I want to create issues for my team (e.g. PURIS), I create a repository with it's name and create the issues there. When I want to organize the work (e.g. on a Kanban Board), I create a project in https://github.com/orgs/eclipse-tractusx/projects?query=is%3Aopen (in Tractus-X Organization) so that its visible for everyone.
For each issue I add in that project I select the PURIS repository which I just created.

Is this correct? :D

**Reply by FaGru3n** [reply]
Hi @PhillipNalesny, baciacally yes with one exeption that for e.g. Puris has already a repository, where you can store your issues and theese issues can be referenced in the project (kanban) board you created. In your working model is it create a draft issue on the board and then convert it to a issue in the puris repository. 

![image](https://github.com/eclipse-tractusx/sig-release/assets/121097161/df00162b-de92-4f25-84f8-0a6fa5be31a8)

Info: Please keep your "Test" issue also updated or close it afterwards if not needed anymore.

**Reply by PhillipNalesny** [reply]
Thank you for your help. I closed the project.
